index.html

```diff
@@ -168,7 +166,7 @@ <h1>Contato</h1>
 		<p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 		</p>
 
-		<hr />
+		<div class="line"></div>
 
 		<address>
 			<a href="mailto:meuemail@mail.com">meuemail@mail.com</a> <br> <br>
```

Comment:
```suggestion
      <a href="mailto:meuemail@mail.com">meuemail@mail.com</a>
``` 

Dica: evite o uso da tag `<br>` Use apenas em último caso quando precisa fazer alguma quebra de linha específica. Neste caso você pode inserir dentro do seu css a propriedade `display block` para a tag a. Assim a quebra de linha é feita de forma mais clean.

Reply:
Entendi, vou mudar isso também.